frontend/GameGUI.py

```diff
@@ -1,20 +1,27 @@
-import pygame as pg
 import sys
-from frontend.BoardGUI import BoardGUI 
+
+import pygame as pg
```

Comment:
Out of curiosity, why did the sys and pygame imports get swapped? Was it just to group the various imports better?

Reply:
Not sure. It couldnt have been alphbetical, maybe `sys` is directly identified as needed to be in top position.